<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="windowTitle">
   <string>Задача</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Заголовок:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="title_input"/>
   </item>
   <item>
    <widget class="QLabel" name="label_desc">
     <property name="text">
      <string>Описание:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="desc_input"/>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>Статус:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="status_input">
     <item>
      <property name="text">
       <string>Ожидает</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>В процессе</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Выполнено</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>